_posts/2019-04-24-gis-wisdom.md

```diff
@@ -0,0 +1,111 @@
+
+On the human side, the more we share, the more we learn.  And when we share, we not only advance the overall knowledge base, we also advance our own. Ideas grow when they are open and shared.  
+
+Explore some of the [existing GIS open data](http://hub.arcgis.com/pages/open-data) or consider contributing to the [Open Street Maps project](https://www.openstreetmap.org/about).
+
+### 5. Editing
+{: .text-left}
+
+`refinement`
+
+The best datasets have lots of edits and editors.  They are continually being reviewed and refined. Think of multi-editing environments or Open Street Maps.  These datasets are so vast and valuable because they are 'live’ and diverse.  Everyone is bringing something unique to the product, which is difficult to produce with a small group of editors.
+
+But it's the refinement process that makes it valuable.  We too can continually refine through learning and sharing and practice and performance.  In turn, this advances our skills and adds to our value.
+
+Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+
+### 6. Domains
+{: .text-left}
+
+`consistency`
+
+Domains are set up in the database to establish known consistently. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well. 
+
+Much like with domains, consistency can lead to reliable and expected results in human performance as well.  Think of athletes or musicians.  It's their consistently (and consequently, the establishment of muscle memory) that make the best so reliable and successful. Sure, repetition plays a role, but without pre-established methods (ie: domains) the results vary.   
+
+Learn more about [attribute domains](https://pro.arcgis.com/en/pro-app/help/data/geodatabases/overview/an-overview-of-attribute-domains.htm) and why they are so important. 
+
+### 7. Topology
+{: .text-left}
+
+`establishing principles`
+
+Topology is all about rules.  Essentially, it allows us to keep our data clean by defining a set of spatial expectations.  We are able to validate when needed and also allow for the inevitable exception.  Topology is so powerful because once you define the rules, you are then free to focus your efforts on other aspects of the data.
+
+Similarly, we also benefit from predefined rules -- or better yet, principles.  For example, think about driving an automobile.  We learn the rules-of-the-road before set foot in the vehicle.  Why?  Because we don’t want to create the rules ‘on-the-fly’.  Knowing the rules-of-the-road in advance, allows us to focus on more enjoyable things, such as conversation or a good audiobook.
```

Comment:
`We learn the rules-of-the-road before set foot in the vehicle.` this sounds funny. Is there a grammar issue?